x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_overview/analytics_overview.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiEmptyPrompt, EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { EuiButtonTo } from '../../../shared/react_router_helpers';
+import { COLLECTION_CREATION_PATH } from '../../routes';
+
+import { EnterpriseSearchAnalyticsPageTemplate } from '../layout/page_template';
+
+import { AnalyticsCollectionTable } from './analytics_collection_table';
+import { AnalyticsCollectionsLogic } from './analytics_collections_logic';
+
+export const baseBreadcrumbs = [
+  i18n.translate('xpack.enterpriseSearch.analytics.collections.breadcrumb', {
+    defaultMessage: 'Analytics Collections',
```

Comment:
It's worthwhile to checkout the [EUI Writing guidelines](https://elastic.github.io/eui/#/guidelines/writing/guidelines#capitalization). We generally don't capitalize things unless it's a feature name.
```suggestion
    defaultMessage: 'Analytics collections',
```